streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1536,7 +1536,7 @@ private Consumer<StreamThread> streamThreadLeaveConsumerGroup(final long remaini
                     removeMembersFromConsumerGroupResult.memberResult(memberToRemove)
                         .get(remainingTimeMs, TimeUnit.MILLISECONDS);
                 } catch (final Exception e) {
-                    log.error("Could not remove static member {} from consumer group {} due to a: {}",
+                    log.error("Could not remove static member {} from consumer group {} due to:",
                         groupInstanceId.get(),
                         applicationConfigs.getString(StreamsConfig.APPLICATION_ID_CONFIG), e);
```

Comment:
I guess you want to log the exception, right? If yes, a third pair of curly braces is missing, isn't it? Method `error()` has signatures
```
public void error(String format, Object... arguments);
```
and 
```
public void error(String msg, Throwable t);
```
There is no signature that takes a number of arguments and then a `Throwable`. I pretty sure that would be incorrect java.

My suggestion is:
```suggestion
                    log.error("Could not remove static member {} from consumer group {} due to: {}",
                        groupInstanceId.get(),
                        applicationConfigs.getString(StreamsConfig.APPLICATION_ID_CONFIG), 
                        e
                    );
```
or
```suggestion
                    log.error("Could not remove static member {} from consumer group {} due to: {}",
                        groupInstanceId.get(), applicationConfigs.getString(StreamsConfig.APPLICATION_ID_CONFIG), e);
```

Logging the stacktrace would also be great!

Reply:
![Screen Shot 2023-04-04 at 5 22 07 PM](https://user-images.githubusercontent.com/8959638/229950738-d000b29e-83b9-490b-a48c-67aafe8fc0dc.png)

Hmmm... I was actually just reacting to an IntelliJ warning -- seems IntelliJ has a bug.

Closing this PR.